File: Backend/src/main/java/com/feedle/feedleapi/Networking/DeleteCommentRequest.java
package com.feedle.feedleapi.Networking;

public class DeleteCommentRequest extends Request {
    private int commentId;

    public DeleteCommentRequest(int commentId) {
        super(RequestType.DeleteComment);
        this.commentId = commentId;
    }

    public int getCommentId() {
        return commentId;
    }
}

File: FeedleDataTier/FeedleDataTier/Network/DeleteCommentRequest.cs
using System.Text.Json.Serialization;
using FeedleDataTier.Models;

namespace FeedleDataTier.Network
{
    public class DeleteCommentRequest : Request
    {
        [JsonPropertyName("commentId")]
        public int CommentId { get; set; }
        public DeleteCommentRequest(int commentId) : base(RequestType.DeleteComment)
        {
            CommentId = commentId;
        }
    }
}